apps/server/lib/lexical/server/boot.ex

```diff
@@ -88,7 +88,8 @@ defmodule Lexical.Server.Boot do
     "1.15.0" => ">= 1.15.3",
     "1.16.0" => ">= 1.16.0",
     "1.17.0-rc" => ">= 1.17.0-rc",
-    "1.17.0" => ">= 1.17.0"
+    "1.17.0" => ">= 1.17.0",
+    "1.80.0" => ">=1.18.0"
```

Comment:
Typo here. We have a long while until v1.80!

```suggestion
    "1.18.0" => ">= 1.18.0"
```

Reply:
Thanks @sorentwo for the correction! Must have been wishful thinking 😉